portfolio/src/main/webapp/assets/js/script.js

```diff
@@ -12,9 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Returns a random item from the array
-function randomPicker(items) {
-    return items[Math.floor(Math.random() * items.length)];
+// Adds a random greeting to the page.
+function addRandomGreeting() {
+    const greetings = ['Welcome', 'Bienvenido', '欢迎', 'Bienvenue', 'Willkommen', 'Benvenuto', 'Bem-vinda', 'желанный'];
+
+    // Pick a random greeting.
+    const greeting = randomPicker(greetings);
+
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+}
+
+/** Creates an <li> element containing text. */
```

Comment:
nit: Can you update the comment to be consistent with the others (eg. //...)

Is this being used yet, or is this for future work?